### PR TITLE
feat(trace viewer): small improvements

### PR DESCRIPTION
**Pull request by dgozman**
- Show logs.
- Show errors.
- Highlight actions.

### CODE REVIEW

File: src/cli/traceViewer/web/ui/timeline.css [hunk 1]
```diff
@@ -72,19 +73,26 @@
   bottom: 0;
   background-color: red;
   border-radius: 3px;
+  --action-color: 'transparent';
+  background-color: var(--action-color);
+}
+
+.timeline-action.selected {
+  filter: brightness(70%);
+  box-shadow: 0 0 0 1px var(--action-color);
```
Comment (JoelEinbinder) [inline src/cli/traceViewer/web/ui/timeline.css:82]: This renders differently from an outline??

```suggestion
  outline: 1px solid var(--action-color);
```

**Review by JoelEinbinder — APPROVED** [review]

File: src/cli/traceViewer/web/ui/timeline.css (hunk 1 above)
Comment (dgozman) [reply src/cli/traceViewer/web/ui/timeline.css:82]: Yep, it respects border-radius!

File: src/cli/traceViewer/web/ui/timeline.tsx [hunk 2]
```diff
@@ -21,43 +21,78 @@ import { FilmStrip } from './filmStrip';
...
+    const time1 = positionToTime(measure.width, boundaries, x - 5);
+    const time2 = positionToTime(measure.width, boundaries, x + 5);
+    let entry: ActionEntry | undefined;
+    let distance: number | undefined;
+    for (const e of actionEntries) {
+      const left = Math.max(e.action.startTime!, time1);
+      const right = Math.min(e.action.endTime!, time2);
+      const middle = (e.action.startTime! + e.action.endTime!) / 2;
+      const d = Math.abs(time - middle);
+      if (left <= right && (!entry || d < distance)) {
```
Comment (JoelEinbinder) [inline src/cli/traceViewer/web/ui/timeline.tsx:68]:
```suggestion
      if (left <= right && (!entry || d < distance!)) {
```

Comment (dgozman) [reply src/cli/traceViewer/web/ui/timeline.tsx]: 👍

**Pull request merged**